src/Sarif/Visitors/SarifVersionOneToCurrentVisitor.cs

```diff
@@ -715,7 +715,7 @@ internal Region CreateRegion(RegionVersionOne v1Region)
                 region = new Region
                 {
                     ByteLength = v1Region.Length,
-                    ByteOffset = v1Region.Offset,
+                    ByteOffset = v1Region.Offset == 0 ? -1 : v1Region.Offset,
```

Comment:
>v1Region.Offset [](start = 33, length = 15)

what if byteLength > 0? in this case you should be preserving a byteOffset of 0, isn't that true?